README.md

```diff
@@ -13,7 +13,7 @@ frameworks for languages such as Python, Java, and Node. For details, see the
 ## Streaming Support
 gRPC-web currently supports 2 RPC modes:
 - Unary RPCs ([example](#make-a-unary-rpc-call))
-- Server-side Streaming RPCs ([example](#server-side-streaming)) (NOTE: Only when [`grpcwebtext`](#wire-format-mode) mode is used.)
+- Server-side Streaming RPCs ([example](#server-side-streaming)) (NOTE: Only when [`grpcwebtext`](#wire-format-mode) mode is used, or when [Nginx](#ecosystem) is used)
```

Comment:
This is a surprising / confusing note. Why does Envoy, the gRPC-web recommended proxy, not support server-side streaming in binary, if Nginx is able to?

Reply:
Yeah, i can agree.. I was just stating the information i gathered, but not trying to reason if it's "expected" or not.. :)

Reply:
The phrasing here is still confusing, even if it's true - it says only the text format supports streaming, but also that Nginx actually supports streaming regardless of format. _If true_ it would be clearer to say that streaming conceptually works with both formats but the Envoy implementation specifically only supports streaming with the text format.

Reply:
Yeah i agree with you.. i didn't call out Envoy specifically because it's [not the only](https://github.com/grpc/grpc-web?tab=readme-ov-file#proxy-interoperability) proxy that supports grpc web.

Let's get a conclusion on the other comment you have, and then i'll update this text.